Sources/LogicKit/Term.swift

```diff
@@ -35,32 +40,96 @@ public enum Term {
 
     // MARK: EDSL
 
-    public static func lit<T>(_ value: T) -> Term where T: Hashable {
+    public static func lit<T>(_ value: T) -> Term where T: Hashable & CustomStringConvertible {
```

Comment:
Why do you request `T` to be `CustomStringConvertible`?

Reply:
I do not remember why. It can be safely removed.